admin/client/App/screens/List/components/ItemsTable/ItemsTableDragDropZoneTarget.js

```diff
@@ -57,46 +57,39 @@ const dropTarget = {
 		const item = monitor.getItem();
 		item.goToPage = props.page;
 		item.prevSortOrder = item.sortOrder;
-		// if the new page is greater, we will place the item at the beginning of the page
-		// if the new page is less, we will place the item at the end of the page
+		// Work out the new sort order. If the new page is greater, we'll put it at the start of the page, and
+		// if it's smaller we'll put it at the end of the page.
 		item.newSortOrder = (targetPage < page) ? (targetPage * pageSize) : (targetPage * pageSize - (pageSize - 1));
-
 		return item;
 	},
+	/*
+	* TODO Work out if it's possible to implement this in a way that works.
 	hover (props, monitor, component) {
 		if (timeoutID) {
 			return;
 		}
-		const { page, currentPage, drag } = props;
-		const original = drag;
+		const { page, currentPage } = props;
 
 		// self
 		if (page === currentPage) {
 			return;
 		}
+
 		if (monitor.isOver()) {
 			timeoutID = setTimeout(() => {
-				const newIndex = (original.page === page) ? original.index : (currentPage < page) ? 0 : props.pageSize;
-				props.dispatch(setCurrentPage(page));
-				monitor.getItem().index = newIndex;
+				// If user hovers over the target for a while change the page.
+				// TODO Get this working. Currently, it looks like it's going to work, but when you
+				// drop onto a new page, no drop events are fired, and react-dnd doesn't have a way to
+				// manually force them to happen. Not sure what to do here.
```

Comment:
So, firstly it doesn't work in current master anyway, but no - it still doesn't work. 

You used to be able to hover over the link to page two, whilst dragging an item, and the page used to change to page two, and it would have dropped the item at the start of page two. Now you will actually have to release the mouse for this to happen.

I don't actually think it's going to be possible to get this working. When the page changes the event seems to get lost, so when you do release the mouse, the drop event is never fired. What's worse is when you then go to try and drag another item, you get many many errors about the fact that you can't start dragging a new item while you're dragging an old item. 

I'll have another go at some point, but not in this PR. The current behaviour just doesn't work at all, so keen to get this merged pretty quickly, and making this little bit work is going to take quite a big effort, and might not be possible at all. 

TL;DR: No it doesn't work, but it doesn't work in master either, and may not be fixable at all. I will have another go at some point, but that'll come later.